### ISSUE TITLE
is it possible to get which key does player press with editing headless-min.js?

### ISSUE THREAD

**Issue by kadavrahaxball** [haxball/haxball-issues#2006]
for example i want to get callback if a player press the K button. is it possible?

**Comment by basro** [comment]
No, it is not possible since haxball doesn't send all the keys the player presses. Only sends the keys that are relevant to the game.

Please do not edit headless-min.js.
